console/backend/src/main/java/org/frankframework/console/controllers/ServerStatistics.java

```diff
@@ -70,7 +75,14 @@ public ResponseEntity<?> downloadActiveConfigurations(@RequestParam(value = "dat
 	@Description("view all configuration warnings")
 	@GetMapping(value = "/warnings", produces = MediaType.APPLICATION_JSON_VALUE)
 	public ResponseEntity<?> getServerConfiguration() {
-		return frankApiService.callSyncGateway(RequestMessageBuilder.create(BusTopic.APPLICATION, BusAction.WARNINGS));
+		ResponseEntity<?> response = frankApiService.callSyncGateway(RequestMessageBuilder.create(BusTopic.APPLICATION, BusAction.WARNINGS));
+
+		String cachedMessage = messageCacheStore.get(frankApiService.getMemberTarget(), BusTopic.APPLICATION.toString());
+		String responseBody = (String)response.getBody();
+		if (cachedMessage == null && responseBody != null) {
+			messageCacheStore.put(frankApiService.getMemberTarget(), BusTopic.APPLICATION.toString(), responseBody);
+		}
+		return response;
```

Comment:
Is dit echt nodig? Kan je niet in de frontend of de websocket ophalen of de api aanroepen?

Reply:
Op dit moment roept de console eerst de api aan tijdens initializatie om zo snel mogelijk de warnings te tonen, daarna wordt er over elke 60 seconde een websocket bericht verstuurd om deze te updaten. Dus we doen beide al eigenlijk.
Omdat de api endpoint eerst niet de cache zette krijg je dus de duplicate warnings